README.md

```diff
@@ -257,47 +223,49 @@ const x = #{ __proto__: foo }; // SyntaxError, __proto__ identifier prevented by
 const y = #{ "__proto__": foo }; // valid, creates a record with a "__proto__" property.
 ```
 
-#### Runtime verification
+Concise methods are disallowed in Record syntax.
+
+```js
+#{ method() { } }  // SyntaxError
+```
 
-At runtime, attempting to create a `Record` with a `symbol` key is a `TypeError`.
+#### Runtime errors
 
-At runtime, attempting to create a `Record` or `Tuple` that contains any type except the following: `Record`, `Tuple`, `string`, `number`, `symbol`, `boolean`, `bigint`, `undefined`, or `null` is a `TypeError`.
+Records may only have String keys, not Symbol keys, due to the issues described in [https://github.com/tc39/proposal-record-tuple/issues/15](#15). Creating a Record with a Symbol key is a `TypeError`.
```

Comment:
since we did provide examples for previous errors, I'd be inclined to add one here:
````suggestion
Records may only have String keys, not Symbol keys, due to the issues described in [https://github.com/tc39/proposal-record-tuple/issues/15](#15). Creating a Record with a Symbol key is a `TypeError`.

```js
const record = #{ [Symbol()]: #{} };
// TypeError: Record may only have string as keys
```
````